test/unit/allowlist/AllowlistVerificationSpec.scala

```diff
@@ -64,6 +70,8 @@ class AllowlistVerificationSpec extends ControllerSpec with BeforeAndAfterEach w
         .download()
         .apply(SessionBuilder.buildRequestWithSessionAndPath("/customs-enrolment-services/subscribe/", defaultUserId))
 
+      println(contentAsString(result))
```

Comment:
Print?

Reply:
Good catch. Thanks! Removed

Reply:
In this file probably it's worth to change a test names (will do it in a separate PR), because tests suggest it will return 401 (actually it's returning 303, because it's a redirect to unauthorised page)